index.html

```diff
@@ -3506,7 +3506,7 @@ <h5>JWT Encoding</h5>
             <p>
 If no explicit rule is specified, <a>properties</a> are encoded in the same way
 as with a standard <a>verifiable credential</a>, and are added to the
-<code>vc</code> <a>property</a> of the JWT. As with all JWTs, the JWS-based
+<code>vc</code> <a>claim</a> of the JWT. As with all JWTs, the JWS-based
```

Comment:
I think this confuses some of the other text, but I'm not going to stand in the way. It's non-normative.